docs/site/DEVELOPING.md

```diff
@@ -144,9 +144,9 @@ test/unit/application.unit.ts
 
 ## API Documentation
 
-We use [strong-docs](https://github.com/strongloop/strong-docs) to generate API
-documentation for all our packages. This documentation is generated when
-publishing new releases to npmjs.org and it's picked up by
+We use [@loopback/tsdocs](https://github.com/strongloop/@loopback/tsdocs) to
```

Comment:
Is it still our intention to allow LB4 developers to build applications and extension in plain JavaScript? If yes, then I think we will eventually need to bring back support for JavaScript. To keep that option open, I am proposing to use a different name than `@loopback/tsdocs`, because that one is tied to TypeScript. Have you considered my proposal for `@loopback/apidocs`?

Reply:
I rename the module as `tsdocs` to make it clear that only TS is supported. If we want to keep JS support, I would rather not to import it into our monorepo as it adds overhead and slows down the build.

Reply:
Fair enough. We can start worrying about JS support when it becomes relevant.